src/ThemedWrappers.h

```diff
@@ -105,6 +109,12 @@ class ThemedButtonWrapper final :
       return mBitmapIndex;
    }
 
+   void SetBitmapFocusIndex(int index)
+   {
```

Comment:
Does this function need to have out of bounds check, especially before calling theTheme.Bitmap with an invalid index?

Reply:
There is a check inside theTheme.Bitmap, but I think it's not really neccessary since all bitmap indices have names in the code